src/backend/columnar/columnar_customscan.c

```diff
@@ -541,7 +541,11 @@ ColumnarIndexScanAdditionalCost(PlannerInfo *root, RelOptInfo *rel,
 	 * "anti-correlated" (-1) since both help us avoiding from reading the
 	 * same stripe again and again.
 	 */
+#if PG_VERSION_NUM >= PG_VERSION_16
+	double absIndexCorrelation = fabs(indexCorrelation);
+#else
 	double absIndexCorrelation = Abs(indexCorrelation);
```

Comment:
What's the logic behind using fabs? Can we have compat macro for this, where Abs-> will turn to fabs?